classpath/java/io/BufferedInputStream.java

```diff
@@ -22,7 +22,7 @@ public BufferedInputStream(InputStream in, int size) {
   }
   
   public BufferedInputStream(InputStream in) {
-    this(in, 4096);
+    this(in, 65536);
```

Comment:
Because it can boost the performance a lot if the underlying inputstream delivers data fast enough. This is an experience i made some years ago in a project where we transferred gigabytes of data around the network. On slow connections it wastes only a small memory amount, but is still as fast as before.